packages/cnpj-gen/src/CnpjGenerator.php

```
@@ -2,99 +2,149 @@
-
-        if ($actualOptions->isFormatting()) {
-            return $this->format($cnpjGenerated);
-        }
-
-        return $cnpjGenerated;
-    }
-
-    public function getOptions(): CnpjGeneratorOptions
-    {
-        return $this->options;
+        $this->options = $options instanceof CnpjGeneratorOptions
```

PHPStan (2.1.54)

[warning] Readonly property Lacus\BrUtils\Cnpj\CnpjGenerator::$options is assigned outside of the constructor.


(property.readOnlyAssignNotInConstructor)